guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+    "initiatives[:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "722236bd",
+   "metadata": {},
+   "source": [
+    "You can also fetch a particular initiative using its item id."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "b942fb0b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Initiative title:\"Walkability Assessment \" owner:mmajumdar_dcdev>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative2 = hub_premium.initiatives.get('07fad5be49da45bea02c8f5b0876be24')\n",
+    "initiative2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1e374ac",
+   "metadata": {},
+   "source": [
+    "### Update an Initiative\n",
+    "\n",
+    "You can update certain initiative properties using the `update` method on the initiative object. The list of properties that can be updated using this method can be found [here](https://developers.arcgis.com/python/api-reference/arcgis.gis.toc.html#arcgis.gis.Item.update)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "e985d147",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Create your own initiative by combining existing applications with a custom site. Use this initiative to form teams around a problem and invite your community to participate.'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative1.snippet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "e24a96b1",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative1.update(initiative_properties={\"snippet\":\"Test change of snippet\"})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "24da6316",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Test change of snippet'"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative1.snippet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "170d00cc",
+   "metadata": {},
+   "source": [
+    "### Delete an Initiative"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "bd874dce",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative1.delete()"
+   ]
```

Comment:
* I think it would help to explain what gets deleted or link to documentation that explains what happens when you delete an initiative - what happens to the application item, initiative item, core team group, content group? What happens to any items shared with these groups upon deletion of an initiative?